app/components/shared/main-navigation.html

```diff
@@ -1,5 +1,8 @@
 <ul>
     <li>
+        <a href="#" [routerLink]="['/accounts', selectedId]" routerLinkActive="active"><i class="fa fa-th fa-lg"></i><span>Dash</span></a>
```

Comment:
Fixed. Was an issue with the top nav bar not being consistent with sending the account name to the rest of the app.